src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/QueueIntegrationTest.kt

```diff
@@ -362,22 +331,24 @@ abstract class QueueIntegrationTest : IntegrationTest() {
       .expectBody().json(fileAssert.readResourceAsText())
   }
 
-  protected fun getOffenderBookingTemplate(): OffenderBooking {
-    return gson.fromJson("/templates/booking.json".readResourceAsText(), OffenderBooking::class.java)
-  }
+  protected fun getOffenderBookingTemplate(): OffenderBooking =
+    gson.fromJson("/templates/booking.json".readResourceAsText(), OffenderBooking::class.java)
+
+  fun PrisonerBuilder.toPrisoner(): Prisoner = PrisonerB(ob = toOffenderBooking(), incentiveLevel = toIncentiveLevel(), null)
```

Comment:
this is a bit naff - currently having to go round the houses to build a prisoner via an offender booking and then translation - will fix later